implementations/rust/ockam/ockam_command/src/util/api.rs

```diff
@@ -210,6 +210,39 @@ pub(crate) fn start_authenticated_service(addr: &str) -> Result<Vec<u8>> {
     Ok(buf)
 }
 
+pub(crate) mod credentials {
+    use super::*;
+    use hex::FromHexError;
+    use ockam_api::nodes::models::credentials::{
+        GetCredentialRequest, PresentCredentialRequest, SetAuthorityRequest,
+    };
+
+    pub(crate) fn present_credential(
+        to: &MultiAddr,
+        oneway: bool,
+    ) -> RequestBuilder<PresentCredentialRequest> {
+        let b = PresentCredentialRequest::new(to, oneway);
+        Request::builder(Method::Post, "/node/credentials/actions/present").body(b)
```

Comment:
```suggestion
        Request::post("/node/credentials/actions/present").body(b)
```

Reply:
Too bad this better approach wasn't propagate everywhere when added, now the old approach is copy&pasted from different places. I'll fix that